src/js/view/controls/components/timeslider.js

```diff
@@ -96,12 +96,18 @@ class TimeSlider extends Slider {
             .change('buffer', this.onBuffer, this)
             .change('streamType', this.onStreamType, this);
 
+
+        this.el.setAttribute('tabindex', '0');
+        this.el.setAttribute('role', 'slider');
+        this.el.setAttribute('aria-label', 'Time Slider');
```

Comment:
Seems like we could make things a lot of space by adding a static `setAttribute` helper .

Reply:
Could you clarify

Reply:
The three lines above will not be minified at all. This will:
```js
function setAttribute(element, attribute, value) {
    element.setAttribute(attribute, value);
}
....
        const element = this.el;
        setAttribute(element, 'tabindex', '0');
        setAttribute(element, 'role', 'slider');
        setAttribute(element, 'aria-label', 'Time Slider');
```

To something more like
```js
function a(b, c, d) {
    b.setAttribute(c, d);
}
....
        var x = this.el;
        a(x, 'tabindex', '0');
        a(x, 'role', 'slider');
        a(x, 'aria-label', 'Time Slider');
```